mintpy/ifgram_inversion.py

```diff
@@ -372,11 +373,11 @@ def estimate_timeseries(A, B, y, tbase_diff, weight_sqrt=None, min_norm_velocity
         return ts, inv_quality, num_inv_obs
 
     # check 2 - matrix invertability (for WLS only because OLS contains it already)
-    if weight_sqrt is not None:
-        try:
-            linalg.inv(np.dot(B.T, B))
-        except linalg.LinAlgError:
-            return ts, inv_quality, num_inv_obs
+    #if weight_sqrt is not None:
+    #    try:
+    #        linalg.inv(np.dot(B.T, B))
+    #    except linalg.LinAlgError:
+    #        return ts, inv_quality, num_inv_obs
```

Comment:
I remember our offline communication about this. Could you please add the comments here in the code also, so that we could keep track of the reasoning behind this change?